extensions/azurecore/src/extension.ts

```diff
@@ -163,7 +163,7 @@ export async function activate(context: vscode.ExtensionContext): Promise<azurec
 		getSqlServers(account: azurecore.AzureAccount,
 			subscriptions: azurecore.azureResource.AzureResourceSubscription[],
 			ignoreErrors: boolean): Promise<azurecore.GetSqlServersResult> {
-			return azureResourceUtils.runResourceQuery(account, subscriptions, ignoreErrors, `where type == "${azurecore.azureResource.AzureResourceType.sqlServer}"`);
+			return azureResourceUtils.runResourceQuery(account, subscriptions, ignoreErrors, `where type == "${azurecore.azureResource.AzureResourceType.sqlServer}" and kind != "v12.0,analytics"`);
```

Comment:
This should reference serversQuery directly

Reply:
This query won't include the synapse servers, which may or may not be intentional, due to the query being split. The only thing this is used in is the getAvailableSqlServers api call. If we want it to list the Synapse servers again even with incorrect properties, we'll have to use the old string here.